indico/modules/events/papers/templates/management/_paper_assignment_list.html

```diff
@@ -0,0 +1,157 @@
+                                {% elif item.id == 'track' %}
+                                    <td class="i-table"
+                                        data-searchable="{{ contrib.track.title | lower }}">
+                                        <span class="vertical-aligner">
+                                            {%- if contrib.track -%}
+                                                {{ contrib.track.title }}
+                                            {%- else -%}
+                                                {% trans %}No track{% endtrans %}
+                                            {%- endif -%}
+                                        </span>
+                                    </td>
+                                {% elif item.id == 'session' %}
+                                    <td class="i-table"
+                                        data-searchable="{{ contrib.session.title | lower }}">
+                                        <span class="vertical-aligner">
+                                            {%- if contrib.session -%}
+                                                {% trans title=contrib.session.title -%}
+                                                    {{ title }}
+                                                {%- endtrans %}
+                                            {%- else -%}
+                                                {% trans %}No session{% endtrans %}
+                                            {%- endif -%}
+                                        </span>
+                                    </td>
+                                {% elif item.id == 'type' %}
+                                    <td class="i-table"
+                                        data-searchable="{{ contrib.type.name | lower }}">
+                                        <span class="vertical-aligner">
+                                            {%- if contrib.type -%}
+                                                {{ contrib.type.name }}
+                                            {%- else -%}
+                                                {% trans %}n/a{% endtrans %}
+                                            {%- endif -%}
+                                        </span>
+                                    </td>
+                                {% endif %}
+                            {% endfor %}
+                            <td class="i-table revision-column">
+                                <span class="vertical-aligner">
+                                    {%- if contrib.paper_revisions -%}
+                                        {{ contrib.paper_revisions | length }}
+                                    {%- endif -%}
+                                </span>
+                            </td>
+                            <td class="i-table person-row-cell"
+                                data-searchable="{{ contrib.paper_judges|map(attribute='name')|join(', ')|lower }}">
```

Comment:
`|join(', ', attribute='name')` - no need  for the separate `|map()`

http://jinja.pocoo.org/docs/dev/templates/#join